src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,78 @@ metadata_model(
     path = "$PKG.LinearCountRegressor"
 )
 
+const DOC_LINEAR_REGRESSION = "TODO"
+"""
+$(MMI.doc_header(LinearRegressor))
+`LinearRegressor` implements the $DOC_LINEAR_REGRESSION.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: any table of input features (eg, a `DataFrame`) whose columns
+  each have one of the following element scitypes: `Continuous`,
+  `Count`, or `<:OrderedFactor`; check column scitypes with `schema(X)`
+
```

Comment:
This model only supports `Continuous` input features. You can see this from it's metadata entry (traits):

```julia
metadata_model(LinearRegressor,
               input   = Table(Continuous),
               target  = AbstractVector{Continuous},
               weights = false,
               descr   = LR_DESCR,
               path    = "$PKG.LinearRegressor"
               )
```

Reply:
Ah gotta love self documenting code

Reply:
should be resolved in latest commit